CHANGELOG.md

```diff
@@ -3,6 +3,8 @@ This page documents production updates to Cloud Tools for IntelliJ. You can chec
 
 ## 18.1.2
 
+You can now add client libraries to your project for accessing Google Cloud APIs. Just select 'Tools > Google Cloud Tools > Add Cloud Libraries ...'.
```

Comment:
My only concern is for App Engine users.   Java 7 won't work, and Java 8 will be a bad experience.

Reply:
Fair point. Do you suggest not including this or rewording it to make it clear?